<!-- -*- mode: XML -*- -->
<launch> 

	<node name="line_follower" type="line_follower.py" pkg="finalProject2" output="screen" >
		<param name="plan_topic" type="string" value="/world_pose_array"/>
		<param name="pose_topic" type="string" value="/pf/viz/inferred_pose" />
		<param name="plan_lookahead" type="int" value="0" />
		<param name="translation_weight" type="double" value="0.88" />
		<param name="rotation_weight" type="double" value="0.12" />
		<param name="kp" type="double" value="2.0" />
		<param name="ki" type="double" value="0.0" />
		<param name="kd" type="double" value="0.15" />
		<param name="error_buff_length" type="int" value="10" />
		<param name="speed" type="double" value="0.75" />
	</node>

</launch>
